ports/sqlpp11-connector-mysql/vcpkg.json

```
@@ -1,27 +1,37 @@
 {
   "name": "sqlpp11-connector-mysql",
-  "version-string": "0.29",
-  "port-version": 2,
-  "description": "A C++ wrapper for MySQL meant to be used in combination with sqlpp11.",
+  "version": "0.61",
```

```suggestion
  "version": "0.29",
```
Just use its own version, and don't need to update it later.

---

Note: This port is empty now. It is not version 0.29 of upstream.
And so, also the description and license fields are invalid w.r.t. to the absence of contents.

---

Because of this, we should leave it as it is before setting it to an empty port

---

This would create conflicts, and it is not how vcpkg is meant to work: Every checkout shall be a fully consistent universe. Either the implementation is in one port, or in another. 
And if some manifest depends on 0.29, it will get upstream's 0.29. Even if it is an empty port in 0.61.
(But this assumes that upstream development of the separate connector really was stopped.)

---

Make sense